### DISCUSSION TITLE
Error during trying to connect

### DISCUSSION THREAD

**Discussion by candanildeniz** [dbeaver/dbeaver#35709] [Q&A] [answered]
### Description

Hi,
I am encountering a problem when I try to connect to the database of the company. when I click on the connect button, it firstly pops a window saying MySQL driver files are missing, but when I try to "download " the file it gives me another error message as

"Network error: Network unavailable
java.io.IOException:Exception during a connection to https://repo1.maven.org

Reason:
Network unavailable
java.io.IOException:Exception during a connection to https://repo1.maven.org"
AND

"Can't create driver instance (class 'com.mysql.cj.jdbc.Driver').
  Error creating driver 'MySQL' instance.
Most likely required jar files are missing.
You should configure jars in driver settings.

Reason: can't load driver class 'com.mysql.cj.jdbc.Driver'
    com.mysql.cj.jdbc.Driver
    com.mysql.cj.jdbc.Driver
"
I've googled this error and tried all available solutions but none of them helps. Does anyone know what else I can do?

Thanks in advance!

### DBeaver Version

24.2.1.202

### Operating System

Windows 10 Enterprise

### Additional context

screen shot attached
<img width="377" alt="DBeaver" src="https://github.com/user-attachments/assets/1ae942e0-1dfb-4ec1-acdc-90c23027d92c">

**Comment by E1izabeth** [comment]
Are you under the firewall or in a corporate network?

**Comment by E1izabeth** [comment, marked as answer]
As a workaround you can try this:
1. Download the driver files manually
2. Open Connection configuration and press Driver Settings
![image](https://github.com/user-attachments/assets/a92c020b-d939-40d9-b5d5-c5a744d97eba)
3. Choose the libraries that can't be downloaded. They're either red or blue for you, and delete them with the Delete button
![image](https://github.com/user-attachments/assets/83a50486-a4b4-49ae-bf22-f490a9668855)
4. Use add file or add folder to add driver files that you downloaded manually

**Comment by candanildeniz** [comment]
> Are you under the firewall or in a corporate network?

yes Elizabeth

**Comment by candanildeniz** [comment]
ok, thanks for your support, issue resolved

**Reply by hpoettker** [reply]
I'm facing the same issue and work around it by downloading the driver manually as described above.

Did you find the root cause why the error appears in the first place?
